src/confirm/views/app.vue

```diff
@@ -223,14 +243,13 @@ export default {
         this.error = error;
       } else {
         this.error = null;
-        this.dependencyOK = true;
         this.installable = true;
         this.message = null;
         return true;
       }
     },
     close() {
-      sendCmd('TabClose');
+      sendCmdDirectly('TabClose');
```

Comment:
`TabClose` seems to work only with `sendCmd`, because the handler depends on `tabId` that is only sent with `sendCmd`.

Reply:
Indeed! I've fixed sendCmdDirectly instead so it automatically switches to sendCmd. I've added a reminder in the function's description so hopefully this will make it less fragile when we add commands in the future.